backendprocesses.cpp

```diff
@@ -100,8 +100,25 @@ void BackendProcesses::threadProcedure()
     all_bytes_in_minute.push_back(bytes);
     all_bytes_in_minute.push_back("</bsr>");
 
+    // determine base path
```

Comment:
Put this in startThread function or constructor since this only need to be ran once

Reply:
Good call, will do